### ISSUE TITLE
As a user, I want to perform a query and download all the products in one API call

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/pds-api#271]
### Checked for duplicates

No - I haven't checked

### 💪 Motivation

...so that I can [why do you want to do this?]

### Acceptance Criteria

**Given** 
**When I perform** 
**Then I expect**

**Comment by scholes-ds** [comment]
I'm not sure this one is necessary. This might require some special streaming code because Geo has a few data files that are over 2TB! Example: https://pds-geosciences.wustl.edu/grail/grail-l-lgrs-5-rdr-v1/grail_1001/shbdr/

**Comment by msbentley** [comment]
Yeah, I think this is likely a dangerous one :-D When it comes to convenience wrappers and scripts, something that accepts a query and returns the list of download URLs, or spits out a curl/wget script to retrieve the data would probably suffice?
